CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
-cmake_minimum_required(VERSION 2.8)
-project(chrome-pak C)
+cmake_minimum_required(VERSION 3.0)

```

Comment:
Would upgrading minimum version of cmake a good idea? Are there any features used in this pr require to do this? Maybe the cmake version or the existance of the policy could be checked before applying a policy.

Link: <https://cmake.org/cmake/help/v2.8.12/cmake.html#command:if>